client/controllers/curatorInbox/curatorSourceDetails.coffee

```diff
@@ -69,16 +69,17 @@ Template.curatorSourceDetails.onRendered ->
           $title.tooltip('hide').attr('data-original-title', '')
         else
           $title.attr('data-original-title', title)
-      @subscribe 'ArticleIncidentReports', source.url
-      if source.enhancements?.dateOfDiagnosis
-        instance.incidentsLoaded.set(true)
-      else
-        Meteor.call 'getArticleEnhancementsAndUpdate', source, (error, enhancements)=>
-          if error
-            notify('error', error.reason)
-          else
-            source.enhancements = enhancements
+      @subscribe 'ArticleIncidentReports', source.url,
+        onReady: ->
```

Comment:
I tracked down the issue causing incidents not to load. If an incident is created without annotations it will trigger an error in the sort function.

I still think there is a problem with the way the onReady callback is being used here in that it leads to an extra server round trip before article processing can begin.